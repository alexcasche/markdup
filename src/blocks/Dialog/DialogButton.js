import { Button } from 'elements'
import { tint } from 'polished'

const DialogButton = Button.extend`
  display: inline-block;
  padding: 10px 18px;
  background-color: ${props => tint(0.78, props.theme.colors.black)};
  color: ${props => tint(0.65, props.theme.colors.grey)};
  font-size: 14px;
  border-radius: 2px;
  margin-left: 15px;
  &:hover {
    background-color: ${props => tint(0.74, props.theme.colors.black)};
    color: ${props => tint(0.25, props.theme.colors.grey)};
  }
`

export default DialogButton
