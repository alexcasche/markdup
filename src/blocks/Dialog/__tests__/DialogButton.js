import DialogButton from '../DialogButton'

describe('DialogButton Element', () => {
  it('renders correctly', () => {
    const tree = render(<DialogButton />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
