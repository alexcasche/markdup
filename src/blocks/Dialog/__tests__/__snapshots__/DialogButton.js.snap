// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogButton Element renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: none;
  outline: none;
  font-family: "IBM Plex Sans Condensed",Helvetica,Arial,sans-serif;
  font-weight: 500;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  text-transform: uppercase;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 0.15s ease-in-out;
  transition: all 0.15s ease-in-out;
  display: inline-block;
  padding: 10px 18px;
  background-color: #4e5051;
  color: #c8c9cd;
  font-size: 14px;
  border-radius: 2px;
  margin-left: 15px;
}

.c0:hover {
  cursor: pointer;
}

.c0:hover {
  background-color: #57595a;
  color: #eaeaec;
}

<button
  className="c0"
/>
`;
